src/transformers/models/perceiver/modeling_perceiver.py

```diff
@@ -2866,17 +2866,15 @@ def __init__(self, modalities: Mapping[str, PostprocessorType], input_is_dict: b
         self.modalities = nn.ModuleDict(modalities)
         self.input_is_dict = input_is_dict
 
-    def forward(
-        self, inputs: torch.Tensor, pos: Optional[torch.Tensor] = None, modality_sizes=None
-    ) -> Mapping[str, torch.Tensor]:
+    def forward(self, inputs: torch.Tensor, modality_sizes=None) -> Mapping[str, torch.Tensor]:
```

Comment:
I'm afraid we can't simply remove pos here, due to the fact that this class (`PerceiverMultimodalPostprocessor`) is a public one. Hence, people may have already used this method.

We can only remove pos arguments inside methods which aren't public.

Reply:
I've reverted the removed `pos` args in the public methods.